app/models/user.rb

```diff
@@ -19,6 +20,10 @@ def name=(name)
     self.first_name, self.last_name = name.split(' ', 2)
   end
 
+  def color
+    super || Colorize.rgb(display_name)
```

Comment:
Yes, I think it's a good idea. And we should also make sure the colors are unique.